### ISSUE TITLE
Document Prometheus upgrade process and frequency

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/deploy#108]

**Comment by jpmckinney** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Prometheus follow-up documentation and configuration](https://trello.com/c/PAZ3SMit/189-prometheus-follow-up-documentation-and-configuration)

**Comment by jpmckinney** [comment]
I see these variables in the private Pillar file control the versions that are installed:

```
server_prometheus_version
server_alertmanager_version
node_exporter_version
server_blackbox_exporter_version
```

From a quick glance, it seems like Prometheus follows semantic versioning, so only major upgrades would be backwards incompatible. The release notes mention such changes. Presently, it only looks like the next version of [node exporter](https://github.com/prometheus/node_exporter/releases) will have such chnages.

I assume all most recent versions are mutually compatible.

Based on the frequency of releases, I figure an annual upgrade is sufficient.

If all that's correct, we can write it up here: https://ocdsdeploy.readthedocs.io/en/latest/deploy/prometheus.html

**Comment by odscjames** [comment]
> Presently, it only looks like the next version of node exporter will have such changes.

Just to note none of those breaking changes should affect us (so far).

**Comment by jpmckinney** [comment]
Closed via #120.
